modules/dxp/apps/osb/osb-site-initializer/osb-site-initializer-customer-portal/extra/remote-app/src/common/components/HighPriorityContacts/SetupHighPriorityContact.jsx

```diff
@@ -49,7 +51,13 @@ const SetupHighPriorityContact = ({
 				name: `${filter}`,
 				role: getContactRoleByFilter(filter),
 			},
-			filterRequest: SearchBuilder.eq('contactsCategory', _filter),
+			filterRequest: `${SearchBuilder.eq(
+				'contactsCategory',
+				_filter
+			)} and ${SearchBuilder.eq(
+				'r_accountEntryToHighPriorityContacts_accountEntryERC',
+				project.accountKey
+			)}`,
```

Comment:
It's possible to to that with SeachBuilder constructor

```suggestion
			filterRequest:  new SearchBuilder()
			    .eq('contactsCategory', _filter)
			    .and()
			    .eq('r_accountEntryToHighPriorityContacts_accountEntryERC', project.accountKey)
			    .build(),
```

Reply:
Nice!